### ISSUE TITLE
plot,section-method ignoring contourLabels

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1903]
Below is how it will look, when I push the new code to GH.  With the CRAN version, though, the bottom panel has the same labels as the top panel.

``` r
library(oce)
#> Loading required package: gsw
data(section)
section <- handleFlags(section)
source("~/git/oce/R/sw.R")
source("~/git/oce/R/ctd.R")
source("~/git/oce/R/section.R")

# 1. Specified contour levels
# Top: automatic labels; bottom: user-specified labels.
par(mfrow=c(2,1))
plot(section, which="temperature",
    contourLevels=seq(5,15,5))
plot(section, which="temperature",
    contourLevels=seq(5,15,5),
    contourLabels=c("cold","intermediate","warm"))
```

![](https://i.imgur.com/DsTNqKL.png)

<sup>Created on 2021-12-31 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by dankelley** [comment]
I'll leave this issue open until I push to GH, perhaps today or tomorrow.

**Comment by dankelley** [comment]
Fixed in commit bbaadfa9b8a52d1f6435d878eaa6d512f4a14e66 of the "develop" branch.
